### PR TITLE
updating locating to downlod the plugin

### PR DESCRIPTION
**Pull request by bam241**
using the news release location to download the plugin

### CODE REVIEW

File: doc/install/plugin.rst [hunk 1]
```diff
@@ -33,4 +33,4 @@ Windows install
 A plugin for Windows is currently being developed; it will be posted to the
 DAGMC plugins page when it is ready.
 
-..  _DAGMC_plugins: https://go.wisc.edu/svalinn-cubit-plugin
+..  _DAGMC_plugins: https://github.com/svalinn/Cubit-plugin/releases/tag/0.1.0
```
Comment (gonuke) [inline doc/install/plugin.rst]: Maybe we should just reference the release page `https://github.com/svalinn/Cubit-plugin/releases` without a specific release, which will allow this to be generic across future releases.

**Review by gonuke — CHANGES_REQUESTED** [review]
I wonder if we should revise our mention of the "DAGMC Plugin" to be the "Cubit Plugin" or the "Svalinn Plugin for Cubit"? or something???

**Comment by bam241** [comment]
@gonuke I applied the suggested change, anything else ?

**Comment by gonuke** [comment]
I had a general recommendation to change how we refer to the plug-in, but maybe this is a different issue.

For DAGMC people, it is a Cubit plugin.  For Cubit people, it is a DAGMC plugin.  Since it includes mcnp2cad, it is actually a Svalinn plugin(?)....

**Review by gonuke — APPROVED** [review]

**Comment by gonuke** [comment]
Thanks @bam241 - we'll tackle the generic naming convention in the future

**Comment by gonuke** [comment]
Now someone needs to rebuild the website?

**Pull request merged**